src/components/ViewDao/ViewDaoContainer.tsx

```diff
@@ -85,13 +85,13 @@ class ViewDaoContainer extends React.Component<IProps, IState> {
     return (
       <div className={css.outer}>
         <BreadcrumbsItem to={"/dao/" + dao.address}>{dao.name}</BreadcrumbsItem>
-        <DaoSidebar address={dao.address} />
+        <DaoSidebar {...this.props} dao={dao} />
```

Comment:
i doubt you want to send all the ViewContainer's props to the DaoSidebar

Reply:
you just need the dao right?

Reply:
also need `currentAccountAddress`.  Should I break that out instead of sending `props`?

Reply:
I guess either way is fine, though in this case with just one prop needed its nicer to spell out what's happening

Reply:
I agree, done.